CMakeLists.txt

```diff
@@ -1352,6 +1352,16 @@ if(WIN32)
 endif(WIN32)
 
 ######################################
+# Remote Packet Capture Support
+######################################
+
```

Comment:
Shouldn't you just add `add_subdirectory(rpcapd)` to the existing code that tests `ENABLE_REMOTE`?  (Note that I changed the option to `ENABLE_REMOTE` from `HAVE_REMOTE`; the new name better describes what what the option actually means.)

Reply:
yes. That confused me a little. I just fixed that.